cli/show.h

```diff
@@ -850,4 +851,74 @@ void shm_info()
 	table.print();
 }
 
+void shm_tsc_info()
+{
+	interface::dataPlane dataplane;
+	const auto response = dataplane.get_shm_tsc_info();
+
+	table_t table;
+	table.insert("core id",
+	             "socket id",
+	             "ipc key",
+	             "offset");
+
+	for (const auto& [core, socket, ipc_key, offset] : response)
+	{
+		table.insert(core, socket, ipc_key, offset);
+	}
+
+	table.print();
+}
+
+void shm_tsc_set_state(bool state)
+{
+	interface::dataPlane dataplane;
+	common::idp::updateGlobalBase::request globalbase;
+	globalbase.emplace_back(common::idp::updateGlobalBase::requestType::tsc_state_update,
+	                        state);
+	dataplane.updateGlobalBase(globalbase);
+}
+
+static const std::map<std::string, unsigned int> counter_name_to_offset = {
```

Comment:
Should i use array instead of struct for dataplane::perf::tsc_base_values, to avoid this?